controlplane/src/core/controllers/auth.ts

```diff
@@ -158,6 +164,46 @@ const plugin: FastifyPluginCallback<AuthControllerOptions> = function Auth(fasti
       return insertedSessions[0];
     });
 
+    const orgs = await opts.organizationRepository.memberships({
+      userId,
+    });
+    if (orgs.length === 0) {
```

Comment:
This would prevent people who already joined an organization from bootstraping their personal org.

Reply:
yeah, it should be fine as people who already have accounts cant register, in the future we have a create org button in the ui

Reply:
ok